libs/http/include/http/server.hpp

```diff
@@ -95,9 +95,6 @@ class HTTPServer : public AbstractHTTPServer
         acceptor->close(dummy);
       }
     });
-
-    // TODO (issue 1220): This appears to cause a double free due to a race
-    /* manager_.reset(); */
```

Comment:
I believe this is an invalid setup issue can you mark this issue as closed with this PR